Client/src/pages/sign_in.vue

```diff
@@ -4,14 +4,16 @@
 
     <q-card style="width: 300px; margin-top: 25px;" >
 
-      <q-card-section> 
+      <q-card-section>
         <div class="text-h6">
           Sign in to PantryPal
         </div>
       </q-card-section>
-      
+
       <q-card-section class="q-gutter-sm">
         <q-form
+          action="http://localhost:8000/#/api"
```

Comment:
I don't think we're using the actual form submission action, we can remove this line.